### ISSUE TITLE
Failed to load plugin

### ISSUE THREAD

**Issue by hlaueriksson** [Advaith3600/PowerToys-Run-Currency-Converter#1]
Hi!

After installing the plugin and restarting PowerToys I get this error:

```
Message: Can't create instance for <Currency Converter> in C:\Users\<user>\AppData\Local\Microsoft\PowerToys\PowerToys Run\Plugins\currency-converter\PowerToysRunCurrencyConverter.dll

Exception full name  : System.Reflection.TargetInvocationException
Exception message    : Exception has been thrown by the target of an invocation.
Exception stack trace:
   at System.RuntimeType.CreateInstanceDefaultCtor(Boolean publicOnly, Boolean wrapExceptions)
   at Wox.Plugin.PluginPair.CreatePluginInstance()
Exception source     : System.Private.CoreLib
Exception target site: System.Object CreateInstanceDefaultCtor(Boolean, Boolean)
Exception HResult    : -2146232828

Exception full name  : System.ArgumentException
Exception message    : Customized cultures cannot be passed by LCID, only by name. (Parameter 'culture')
Exception stack trace:
   at System.Globalization.RegionInfo..ctor(Int32 culture)
   at PowerToysRunCurrencyConverter.Main..ctor() in C:\Users\advai\AppData\Local\Microsoft\PowerToys\PowerToys Run\Plugins\PowerToysRunCurrencyConverter\PowerToysRunCurrencyConverter\Main.cs:line 24
   at System.RuntimeType.CreateInstanceDefaultCtor(Boolean publicOnly, Boolean wrapExceptions)
Exception source     : System.Private.CoreLib
Exception target site: Void .ctor(Int32)
Exception HResult    : -2147024809
```

**Comment by Advaith3600** [comment]
Hey @hlaueriksson, thank you for your feedback. A potential fix for this is rolled out in [v1.0.2](https://github.com/Advaith3600/PowerToys-Run-Currency-Converter/releases/tag/v1.0.2). Please try installing the new version.

**Comment by hlaueriksson** [comment]
The fix works like a charm 🚀
